packages/fp-behandling-forstegang-og-revurdering/src/fakta/components/arbeidsforholdInfoPanel/BekreftOgForsettKnapp.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import ElementWrapper from '@fpsak-frontend/shared-components/src/ElementWrapper';
+import VerticalSpacer from '@fpsak-frontend/shared-components/src/VerticalSpacer';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { connect } from 'react-redux';
+import PersonArbeidsforholdPanel from './personArbeidsforholdPanel/PersonArbeidsforholdPanel';
+
+/**
+ * Component: BekreftOgForsettKnapp
```

Comment:
Her burde du kanskje følge formatet på dokumentasjonen i resten av prosjektet:
```suggestion
 * BekreftOgForsettKnapp
 *
 * Presentasjonskomponent: Viser knapp for bekreftelse av aksjonspunkt.
```